test/spec/general.js

```diff
@@ -1,9 +1,9 @@
-/* globals SDK_VERSION */
 define(function(require) {
   var OktaAuth = require('OktaAuth');
   var Q = require('q');
   var util = require('../util/util');
   var _ = require('lodash');
+  var packageJson = require('../../package.json');
```

Comment:
We are, but this verifies that the config.json is written with the correct values.